app/controllers/spree/admin/digital_assets_controller.rb

```diff
@@ -11,11 +11,18 @@ def index
       end
 
       def create
-        @object.assign_attributes(permitted_resource_params)        
+        @object.assign_attributes(permitted_resource_params)
         if @object.folder.nil?
           @object.folder = spree_folder
         end
         if @object.save
+          if params[:image] && image_params.present?
```

Comment:
@sdbradley yes, this should probably return a 422 if the image params are not supplied. I was going by the spec, which does not supply those params in the create test.